tests/api/cart.api.test.js

```diff
@@ -1,41 +1,120 @@
 const { test, expect } = require('@playwright/test')
 
 const { getAuthToken, getProductBody, getProductId, getCartBody, getCartId } = require('../../lib/helpers')
-  
+
 test.describe.parallel('Carts API', () => {
   let authorization
-  
+
   test.beforeAll(async ({ request }) => {
     authorization = await getAuthToken(request)
   })
-  
+
   test('retrieves the list of carts', async ({ request }) => {
     const response = await request.get('/carrinhos')
-    
+
+    await expect(response).toBeOK()
+  })
+
+  test('retrieves an existing cart by its id', async ({ request }) => {
+    const idProduct = await getProductId(request, authorization, getProductBody())
+    const _id = await getCartId(request, authorization, getCartBody(idProduct))
+
+    const response = await request.get(`/carrinhos/${_id}`)
+
     await expect(response).toBeOK()
   })
 
   test('creates a cart successfully', async ({ request }) => {
-    const id = await getProductId(request, authorization, getProductBody())
+    const authorization = await getAuthToken(request)
+    const idProduct = await getProductId(request, authorization, getProductBody())
 
     const response = await request.post('/carrinhos', {
-      data: getCartBody(id),
+      data: getCartBody(idProduct),
       headers: { 'Authorization': authorization }
     })
 
     await expect(response).toBeOK()
 
     const responseBody = JSON.parse(await response.text())
-    
+
     await expect(responseBody.message).toBe('Cadastro realizado com sucesso')
   })
 
-  test('retrieves an existing cart by its id', async ({ request }) => {
+  test('delete an order successfully', async ({ request }) => {
     const idProduct = await getProductId(request, authorization, getProductBody())
-    const _id = await getCartId(request, authorization, getCartBody(idProduct))
-    
-    const response = await request.get(`/carrinhos/${_id}`)
-    
+    await getCartId(request, authorization, getCartBody(idProduct))
+
+    const response = await request.delete('/carrinhos/concluir-compra', {
+      headers: { 'Authorization': authorization }
+    })
+
+    await expect(response).toBeOK()
+
+    const responseBody = JSON.parse(await response.text())
+
+    await expect(responseBody.message).toBe('Registro excluído com sucesso')
+  })
+
+  test('delete an order without a cart', async ({ request }) => {
+    const response = await request.delete('/carrinhos/concluir-compra', {
+      headers: { 'Authorization': authorization }
+    })
+
+    await expect(response).toBeOK()
+
+    const responseBody = JSON.parse(await response.text())
+
+    await expect(responseBody.message).toBe('Não foi encontrado carrinho para esse usuário')
```

Comment:
@uLucasFraga same as the [previous comment](https://github.com/stefanteixeira/demo-playwright-test/pull/34/files#r937202926)